tests/test_packages/test_skills_integration/test_tac.py

```diff
@@ -625,7 +625,8 @@ def test_tac(self):
         ), "Strings {} didn't appear in tac_aea_two output.".format(missing_strings)
 
         check_strings = (
-            "agent registered:",
```

Comment:
Actual reason why the test was failing (i.e. not passing assertions)

Reply:
Why wasn't the correct message displayed in the failures? Is this prevented now?